packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboard.tsx

```diff
@@ -105,6 +107,22 @@ export const CloudPulseDashboard = (props: DashboardProperties) => {
     Boolean(resourceList)
   );
 
+  if (isDashboardApiError) {
+    return (
+      <Grid item xs>
```

Comment:
` <Grid item xs>
  {isDashboardApiError && (
    <ErrorState errorText="Failed to load the dashboard." />
  )}

  {isResourcesApiError && (
    <ErrorState errorText="Failed to fetch resources." />
  )}
</Grid>`